operations/IT/hardware-buy-back-policy.md

```diff
@@ -0,0 +1,22 @@
+# Hardware Buy Back Policy
+
+Staff members have the option to buy back their existing hardware (laptop, desktop, phone, tablet etc.) either when it gets refreshed for a new one (which is on average, 3 years from purchase date - not necessarily 3 years of employment), or when the staff member is offboarding. 
+
+If the staff member has completed one calendar year or more at Mattermost at the time of offboarding, they can opt to keep their device at no cost. If the staff member hasn't completed one calendar year at the time of offboarding or is receiving an early hardware refresh, they have the option to purchase their hardware from Mattermost at current market value.
+
+If the hardware requires a buyout, IT will send the staff member an email with the determined value. Then if the employee decides to move forward with the purchase, our accounting department will reach out with payment information.
+
+If a staff member decides to retain their hardware, they are required to wipe the device, re-install the base operating system, and remove any and all software and configurations that were supplied by Mattermost. You will be required to sign a document acknowledging you they have not taken or retained any confidential Mattermost software or data.
```

Comment:
Will this be incorporated as part of the offboarding or should we include a link to the document for staff members to download and return?

Reply:
HR is providing the document via Docusign they require to sign as a part of the offboarding process.